src/components/control-bar/VolumeMenuButton.js

```diff
@@ -62,10 +65,30 @@ class VolumeMenuButton extends Component {
     });
   }
 
+  getDefaultChildren() {
+    return [
+      <VolumeBar
+        onFocus={this.handleFocus}
+        onBlur={this.handleBlur}
+        {...this.props}
+      />
+    ];
+  }
+
+  getChildren() {
+    const children = React.Children.toArray(this.props.children);
+    const defaultChildren = this.getDefaultChildren();
+    const { className, ...parentProps } = this.props; // remove className
+    return mergeAndSortChildren(defaultChildren, children, parentProps);
+  }
+
```

Comment:
Do we need this? I think the `mergeAndSortChildren` method filters out the children that are equal to any of the ones in defaultChildren, so having this might not be doing anything for us. I also don't think I see us having a use case where we're passing in components that aren't the `<VolumeBar />` into the component. Happy to talk more about this though

Reply:
Mmm...excellent point! I would like to keep it here in case we _might_ add new props. I think you're right that we wouldn't be passing in any components that aren't `<VolumeBar />` -- I was just following a pattern previously set in `<ControlBar />`.